modules/certificate-injection-using-operators.adoc

```diff
@@ -25,3 +25,39 @@ The Operator mounts this ConfigMap into the container's local trust store.
 Adding a trusted CA certificate is only needed if the certificate is not
 included in the {op-system-first} trust bundle.
 ====
+
+Certificate injection is not limited to Operators. The Cluster Network Operator
+is responsible for fulfilling the certificate injection request and will do so
```

Comment:
Maybe s/is responsible for fulfilling the certificate injection request and will do so/injects certificates